halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/openstack/OpenstackProvider.java

```diff
@@ -16,4 +20,19 @@ public ProviderType providerType() {
   public void accept(ConfigProblemSetBuilder psBuilder, Validator v) {
     v.validate(psBuilder, this);
   }
+
+  @Override
+  public  OpenstackBakeryDefaults emptyBakeryDefaults() {
+    OpenstackBakeryDefaults result = new OpenstackBakeryDefaults();
+    // Set defaults over here
+    result.setAuthUrl("http://default.com");
+    result.setFloatingIpPool("default_pool");
+    result.setInsecure(false);
+    result.setNetworkId("xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx");
```

Comment:
These defaults will be set any time someone initializes the bakery options. The way these read, I'm guessing they'd have to be cleared/overwritten each time for this to be useful. If that's the case, just leave the fields blank.